gensim/corpora/_mmreader.pyx

```diff
@@ -188,7 +188,7 @@ cdef class MmReader(object):
         if offset == -1:
             return []
         if isinstance(self.input, string_types):
-            fin, close_fin = utils.smart_open(self.input), True
+            fin, close_fin = utils.open(self.input, 'rb'), True
```

Comment:
@piskvorky Hi!
I guess that this change should update `_mmreader.c`  /  `_mmreader.so` right?
How do I trigger this compilation?
I tried few things with no lack, `c` file changed but the `so` didn't, and I'm still getting:
```
  File "gensim/corpora/_mmreader.pyx", line 191, in gensim.corpora._mmreader.MmReader.docbyoffset
AttributeError: module 'gensim.utils' has no attribute 'smart_open'
```

Reply:
CC @mpenkov what's the process regenerating the compiled modules? Force `pip` re-installation of `gensim`?

Reply:
Running cython on the affected files will also work, if gensim in set up in develop mode.

Reply:
Thanks guys, I generated the file and the tests passed. Can you review again, please?

Reply:
Thank you. I'll have a look closer to the weekend.